store/postgres/src/store.rs

```diff
@@ -641,7 +649,24 @@ impl Store {
     }
 
     fn get_entity_conn(&self, subgraph: &SubgraphDeploymentId) -> Result<e::Connection, Error> {
+        let start = Instant::now();
         let conn = self.get_conn()?;
+        self.get_entity_conn_timers
+            .lock()
+            .unwrap()
+            .entry(subgraph.clone())
+            .or_insert_with(|| {
+                *self
+                    .registry
+                    .new_counter(
+                        format!("{}_get_conn_secs", subgraph),
```

Comment:
Should this be `{}_get_entity_conn_secs` perhaps?

Reply:
done